calibration/default_cal.xml

```diff
@@ -3,20 +3,20 @@
 <root>
 
   <!-- Robot Base -> Left arm base -->
-  <property name="cal_base_to_left_arm_x"     value="0.0" />
-  <property name="cal_base_to_left_arm_y"     value="0.5" />
-  <property name="cal_base_to_left_arm_z"     value="1.1" />
-  <property name="cal_base_to_left_arm_roll"  value="0" />
-  <property name="cal_base_to_left_arm_pitch" value="0" />
-  <property name="cal_base_to_left_arm_yaw"   value="0" />
+  <xacro:property name="cal_base_to_left_arm_x"     value="0.0" />
+  <xacro:property name="cal_base_to_left_arm_y"     value="0.5" />
+  <xacro:property name="cal_base_to_left_arm_z"     value="1.1" />
+  <xacro:property name="cal_base_to_left_arm_roll"  value="0" />
+  <xacro:property name="cal_base_to_left_arm_pitch" value="0" />
+  <xacro:property name="cal_base_to_left_arm_yaw"   value="0" />
 
   <!-- Robot Base -> Right arm base -->
   [0.385121, -0.0570121, 1.15408, 5.57141, 5.42596, 2.4299]
-  <property name="cal_base_to_right_arm_x"     value="0.0" />
-  <property name="cal_base_to_right_arm_y"     value="-0.5" />
-  <property name="cal_base_to_right_arm_z"     value="1.1" />
-  <property name="cal_base_to_right_arm_roll"  value="0.0" />
-  <property name="cal_base_to_right_arm_pitch" value="0.0" />
-  <property name="cal_base_to_right_arm_yaw"   value="0.0" />
+  <xacro:property name="cal_base_to_right_arm_x"     value="0.0" />
+  <xacro:property name="cal_base_to_right_arm_y"     value="-0.5" />
+  <xacro:property name="cal_base_to_right_arm_z"     value="1.1" />
+  <xacro:property name="cal_base_to_right_arm_roll"  value="0.0" />
+  <xacro:property name="cal_base_to_right_arm_pitch" value="0.0" />
+  <xacro:property name="cal_base_to_right_arm_yaw"   value="0.0" />
 
```

Comment:
Are the calibration files used?

Reply:
It does not appear so. It is not (and was never) included by any other xacro/xml file
Should it be included by `kuka_lwr_arm_defs.xml`?

Reply:
Remove it if it's not used. It looks like a bimanual setup config.

Reply:
The file `ias_cal_190809.xml` is also not referenced in any other file. Should I remove the whole folder `calibration`?